test/UnitWOPISaveOnExit.cpp

```diff
@@ -213,7 +209,7 @@ class UnitSaveOnExitSaved : public WopiTestServer
     /// The document is loaded.
     bool onDocumentLoaded(const std::string& message) override
     {
-        LOG_TST("onDocumentLoaded: [" << message << ']');
```

Comment:
isn't that better to have different strings for later debugging?
or `onDocumentLoaded` is already in the message?

Reply:
It is already in the log. This is from before including the function name in the log. In this case it's duplicate and not helpful